plugins/frontend/backstage-plugin-github-insights/src/components/Widgets/ReadMeCard/ReadMeCard.tsx

```diff
@@ -139,33 +84,9 @@ const ReadMeCard = (props: ReadMeCardProps) => {
           maxHeight: `${props.maxHeight}px`,
         }}
       >
-        <MarkdownContent
-          content={b64DecodeUnicode(value.content)
-            .replace(
-              /\[([^\[\]]*)\]\((?!https?:\/\/)(.*?)(\.png|\.jpg|\.jpeg|\.gif|\.webp|\.svg)(.*)\)/gim,
```

Comment:
Previously the readme content was being regex replaced with absute urls. This doesnt work for private content on github because there is no way to authenticate static content in src urls.